### ISSUE TITLE
Error ENOENT: no such file or directory for auth0-lock.js

### ISSUE THREAD

**Issue by Ridermansb** [auth0/auth0-ember-simple-auth#51]
Serving on http://localhost:4200/
The Broccoli Plugin: [SourceMapConcat: Concat: Vendor /assets/vendor.js] failed with:
Error: ENOENT: no such file or directory, stat '/Users/ridermansb/Projects/tmp/auth0EmberStarter/tmp/source_map_concat-input_base_path-JzA3yBgT.tmp/0/bower_components/auth0-lock/build/auth0-lock.js'
    at Error (native)
    at Object.fs.statSync (fs.js:892:18)
    at ConcatWithMaps.keyForFile (/Users/ridermansb/Projects/tmp/auth0EmberStarter/node_modules/ember-cli/node_modules/broccoli-concat/node_modules/broccoli-caching-writer/index.js:87:20)
    at Array.map (native)
    at ConcatWithMaps.CachingWriter._conditionalBuild (/Users/ridermansb/Projects/tmp/auth0EmberStarter/node_modules/ember-cli/node_modules/broccoli-concat/node_modules/broccoli-caching-writer/index.js:109:65)
    at /Users/ridermansb/Projects/tmp/auth0EmberStarter/node_modules/ember-cli/node_modules/broccoli-concat/node_modules/broccoli-caching-writer/node_modules/broccoli-plugin/read_compat.js:61:34
    at lib$rsvp$$internal$$tryCatch (/Users/ridermansb/Projects/tmp/auth0EmberStarter/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:1036:16)
    at lib$rsvp$$internal$$invokeCallback (/Users/ridermansb/Projects/tmp/auth0EmberStarter/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:1048:17)
    at lib$rsvp$$internal$$publish (/Users/ridermansb/Projects/tmp/auth0EmberStarter/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:1019:11)
    at lib$rsvp$asap$$flush (/Users/ridermansb/Projects/tmp/auth0EmberStarter/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:1198:9)

**Comment by adamgarcia4** [comment]
I have the same problem.

**Comment by justAdevTV** [comment]
This worked for me. When you're asked to overwrite your current files, just let it overwrite these:

Overwrite app/routes/application.js? Yes, overwrite
Overwrite app/routes/index.js? Yes, overwrite

**Comment by jdno** [comment]
Sorry @justAdevTV, it still fails for me. I even created a fresh app, followed [Auth0's guide](https://auth0.com/docs/quickstart/spa/ember2js) and still ended up with the same error:

    The Broccoli Plugin: [SourceMapConcat: Concat: Vendor /assets/vendor.js] failed with:
    Error: ENOENT: no such file or directory

**Comment by mrstebo** [comment]
+1 Same issue for me with a fresh app

**Comment by dbinetti** [comment]
There is a warning generated during  install:

```
WARNING: Could not figure out blueprint name from: "auth0/auth0-ember-simple-auth". Please install the addon blueprint via "ember generate <addon-name>" if necessary.
```

so if you just run:

```
ember generate auth0-ember-simple-auth
```

It should add the dependencies.

**Comment by adamgarcia4** [comment]
@dbinetti This seemed to have worked for me!

Also, according to Ryan Chenkie from the Ember Team:

> We've got updates coming to our Ember 2 sample that should be up very soon. We'll let you know here when that happens.

Hopefully this means that they will correct any confusions that we are having.

**Comment by bdavidxyz** [comment]
I have the same issue, the @dbinetti workaround perfectly works.

**Comment by graham-sportsmgmt** [comment]
It would be nice if the statement that fixes this issue (thanks!)...

  ember generate auth0-ember-simple-auth

... is added to the auth0 ember quickstart guide:

https://auth0.com/docs/quickstart/spa/ember2js

**Comment by b00mtastik** [comment]
I was excited to see there was a work around for this issue, but I ended up getting this error:

Failed to execute "git ls-remote --tags --heads https://github.com/kjur/jsrsasign=jsrsasign.git", exit code of #128
fatal: unable to access 'https://github.com/kjur/jsrsasign=jsrsasign.git/': The requested URL returned error: 400

Has anyone come across this issue? Has the git repo URL changed and not updated in the bower packages? Or have I managed to just try this while you guys are changing things around?
